epub34/authoring/index.html

```diff
@@ -8625,7 +8546,7 @@ <h4>Introduction</h4>
 					<p>The <code>text</code> element <code>src</code> attribute references the associated phrase,
 						sentence, or other segment of the EPUB content document by its URL [[url]] reference. The
 							<code>audio</code> element <code>src</code> attribute similarly references the location of
-						the corresponding audio clip and adds the OPTIONAL <a href="#attrdef-smil-clipBegin"
+						the corresponding audio clip and adds the optional <a href="#attrdef-smil-clipBegin"
```

Comment:
```suggestion
						the corresponding audio clip and adds the optional <a href="#attrdef-smil-clipBegin"
```

"optional" is also a keyword in rfc2119; I guess it must be replaced, too. But I am not sure with what...

Reply:
Does it even need saying that these are optional in the introduction? It's just generally explaining that they exist and how to use them. I'd just take the word out.

Reply:
```suggestion
						the corresponding audio clip and adds the <a href="#attrdef-smil-clipBegin"
```